plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/JwtTokenManager.java

```diff
@@ -64,6 +65,8 @@ public class JwtTokenManager {
      * Token validity time(seconds).
      */
     private long tokenValidityInSeconds;
+
+    private JwtParser jwtParser;
```

Comment:
I think it would be better to add volatile and have `jwtParser` initialized only once